.github/workflows/ci.yml

```diff
@@ -8,6 +8,50 @@ env:
   GOPROXY: off
 
 jobs:
+  proto:
+    # TODO: use Windows after merginig https://github.com/containerd/protobuild/pull/51.
```

Comment:
@TBBle Should we use Windows for consistency?

Reply:
#51 got merged! Let me update this PR. I agree that generating Go files from proto files is platform-independent. Running Protobuild on different platforms is something Protobuild has to do. So hcsshim doesn't have to do that.